### ISSUE TITLE
Browser-sync via proxy not staying connected.

### ISSUE THREAD

**Issue by Reklino** [BrowserSync/browser-sync#612]
I'm using browser-sync through gulp and have it set up to proxy a domain served from a VM (vagrant box). I am running `gulp` on the **host** machine. If it helps, the Vagrant box is homestead.

It works sometimes for about 15-45 seconds, but then disconnects. If I refresh, it usually reconnects and works for another 15 seconds. Other times it never works, but instead displays a notification in the top right that says 'Disconnected from BrowserSync'.

When I visit the UI page, a warning appears that says 'Disconnected from BrowserSync'. This warning screen will usually flash on and off a few times before the UI page appears to work properly.

It seems like it just can't keep the connection steady.

There are no errors in the CLI. It continues to watch and notify like nothing is wrong.

Here is my gulp file:
``` javascript
var gulp = require('gulp');
var sass = require('gulp-sass');
var browserSync = require('browser-sync').create();
var reload      = browserSync.reload;


gulp.task('sync', function() {
    browserSync.init({
        proxy: "homestead.app"
    });

    gulp.watch('sass/*.scss', ['sass']);
    gulp.watch("app/**/*.html").on('change', reload);
    gulp.watch("app/**/*.js").on('change', reload);
});


gulp.task('sass', function() {
	return gulp.src('sass/*.scss')
		.pipe(sass({
            includePaths: [
                'node_modules/susy/sass' //required for sass
            ]
        }))
		.pipe(gulp.dest('css'))
		.pipe(reload({stream: true}));

});

gulp.task('default', ['sync']);
```
node v0.12.0
gulp 3.8.11
browser-sync 2.7.0

**Comment by shakyShane** [comment]
fixed in browser-sync 2.7.1 - please update

**Comment by Reklino** [comment]
well that was easy :). Wish I had updated sooner.
